encrypted-media-respec.html

```diff
@@ -2551,18 +2607,12 @@
           <p>This section describes the format of the license release message to be provided via the <a def-id="message-event-message-attribute"></a> attribute of the <a def-id="message"></a> event.</p>
           <p>The format is a JSON object containing the following members:</p>
           <dl>
-            <dt>"keys"</dt>
-            <dd>
-              An array of objects, each object containing the following members:
-              <dl>
-                <dt>"kid"</dt>
-                <dd>The base64url encoding of the octet sequence containing a key ID value.</dd>
-                <dt>"tfirst"</dt>
-                <dd>The <var>first decrypt time</var> for this key ID expressed as a number giving the time, in milliseconds since 01 January, 1970 UTC.</dd>
-                <dt>"tlatest"</dt>
-                <dd>The <var>latest decrypt time</var> for this key ID expressed as a number giving the time, in milliseconds since 01 January, 1970 UTC.</dd>
-              </dl>
-            </dd>
+            <dt>"kids"</dt>
+            <dd>An array of <a def-id="key-id">key IDs</a>. Each element of the array is the base64url encoding of the octet sequence containing the key ID value.</dd>
+            <dt>"firstTime"</dt>
+            <dd>The <a def-id="first-decrypt-time"></a> expressed as a number giving the time, in milliseconds since 01 January, 1970 UTC.</dd>
```

Comment:
This is the only place the definition is used. That's sort of useful, but it doesn't appear we do that in other items in this section, including "kids".